gh-issues-import-ng.py

```diff
@@ -13,496 +13,528 @@
 default_config_file = os.path.join(__location__, 'config.ini')
 config = configparser.RawConfigParser()
 
+
 class state:
-	current = ""
-	INITIALIZING         = "script-initializing"
-	LOADING_CONFIG       = "loading-config"
-	FETCHING_ISSUES      = "fetching-issues"
-	GENERATING           = "generating"
-	IMPORT_CONFIRMATION  = "import-confirmation"
-	IMPORTING            = "importing"
-	IMPORT_COMPLETE      = "import-complete"
-	COMPLETE             = "script-complete"
-	
+    current = ""
+    INITIALIZING = "script-initializing"
+    LOADING_CONFIG = "loading-config"
+    FETCHING_ISSUES = "fetching-issues"
+    GENERATING = "generating"
+    IMPORT_CONFIRMATION = "import-confirmation"
+    IMPORTING = "importing"
+    IMPORT_COMPLETE = "import-complete"
+    COMPLETE = "script-complete"
+
+
 state.current = state.INITIALIZING
 
 http_error_messages = {}
-http_error_messages[401] = "ERROR: There was a problem during authentication.\nDouble check that your username and password are correct, and that you have permission to read from or write to the specified repositories."
-http_error_messages[403] = http_error_messages[401]; # Basically the same problem. GitHub returns 403 instead to prevent abuse.
-http_error_messages[404] = "ERROR: Unable to find the specified repository.\nDouble check the spelling for the source and target repositories. If either repository is private, make sure the specified user is allowed access to it."
+http_error_messages[
```

Comment:
@pepper-jk Why dou you update the existing http_error_messages statement? what for?

Reply:
This was an automated change, I was not aware of. It was probably made to meet the line ending specifications of the [pep8 style guide](https://www.python.org/dev/peps/pep-0008/), but for whatever reason it did not split the string to complete this task. I'll change that.

Reply:
Fixed.

Reply:
Thanks.